rendermodules/dataimport/schemas.py

```diff
@@ -108,7 +107,7 @@ class MakeMontageScapeSectionStackParameters(OutputStackParameters):
     montage_stack = Str(
         required=True,
         metadata={'description':'stack to make a downsample version of'})
-    image_directory = InputDir(
+    image_directory = Str(
```

Comment:
Shouldn't this be an OutputDir?

Reply:
no .. i'm wrong.  But now i'm still confused why InputDir is wrong?  This should prevent the module from running with incorrectly specified input directories? Is there an argschema bug?

Reply:
@fcollman The point of this is that the module has the option of generating the downsample if it does not exist.  We are currently using the functionality in the workflow without using the downsampling strategy proper, so it is failing am a nonexistent `image_directory` even though the module would do what is intended.  `OutputDir` isn't quite right either since it requires the value be writable -- it could be read only if the montage has previously been generated.

Reply:
k... I get it now.  The directory needs to either already exist OR be writeable.  which is neither InputDir or OutputDir.